### DISCUSSION TITLE
Cannot make more than one Dynamo DB cable per serverless aws yaml project

### DISCUSSION THREAD

**Discussion by jonathanalberghini** [serverless/serverless#11719] [General]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest v3 release?

- [X] Yes, I'm using the latest v3 release

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description
here is same problem not answered 
https://forum.serverless.com/t/iamrolestatements-for-multiple-dynamodb-tables/8438

For some reason when I add these roll statements I get the error saying it doesn't have rights for the second table to create it. Does serverless support multiple tables? In an error saying that it does not have rights to describe table
It works with one table but not two and there is another issue in the serverless help Sention that no one has answered with the same exact problem.
CREATE_FAILED: MktgHasOffersLog (AWS::DynamoDB::Table)
Resource handler returned message: “User: arn:aws:sts::924149985142:assumed-role/codebuildv5-CodeBuildStandardRole-1DG17IWSQD2WT/AWSCodeBuild-482aa296-5eb2-4041-a4c9-6f9a78942f67 is not authorized to perform: dynamodb:DescribeTable on resource: arn:aws:dynamodb:us-east-1:924149985142:table/Mktg-HasOffers-Logs-dev because no identity-based policy allows the dynamodb:DescribeTable action (Service: DynamoDb, Status Code: 400, Request ID: D2M4SJM8T0KA7ETBU5FKS224M3VV4KQNSO5AEMVJF66Q9ASUAAJG)” (RequestToken: ecf708b3-58eb-a783-8775-890035cf6fc3, HandlerErrorCode: AccessDenied)

`` iamRoleStatements:

 - Effect: "Allow"
  Action:
    - dynamodb:BatchGet*,
    - dynamodb:DescribeStream,
    - dynamodb:DescribeTable,
    - dynamodb:Get*,
    - dynamodb:Query,
    - dynamodb:Scan,
    - dynamodb:BatchWrite*,
    - dynamodb:CreateTable,
    - dynamodb:Delete*,
    - dynamodb:Update*,
    - dynamodb:PutItem
  Resource:
    - { "Fn::GetAtt": ["MktgHasOffersLog", "Arn"] } # do not rename this
    - { "Fn::GetAtt": ["MktgJourneySingleSend", "Arn"] } # do not rename this
``

    MktgJourneySingleSend: # do not rename this
      Type: AWS::DynamoDB::Table
      DeletionPolicy: Retain
      Properties:
        TableName: ${self:provider.environment.DYNAMODB_TABLE_TS}
        AttributeDefinitions:
          - AttributeName: "PK"
            AttributeType: "S"
          - AttributeName: "SK"
            AttributeType: "S"
        KeySchema:
          - AttributeName: "PK"
            KeyType: "HASH"
          - AttributeName: "SK"
            KeyType: "RANGE"
        BillingMode: "PAY_PER_REQUEST"
    MktgHasOffersLog: # do not rename this
      Type: AWS::DynamoDB::Table
      DeletionPolicy: Retain
      Properties:
        TableName: ${self:provider.environment.DYNAMODB_TABLE_HO}
        AttributeDefinitions:
          - AttributeName: pk
            AttributeType: S
          - AttributeName: sk
            AttributeType: S
          - AttributeName: gs1pk
            AttributeType: S
          - AttributeName: gs1sk
            AttributeType: S
        KeySchema:
          - AttributeName: pk
            KeyType: HASH
          - AttributeName: sk
            KeyType: RANGE
        GlobalSecondaryIndexes:
          - IndexName: gs1
            KeySchema:
              - AttributeName: gs1pk
                KeyType: HASH
              - AttributeName: gs1sk
                KeyType: RANGE
            Projection:
              ProjectionType: ALL
        BillingMode: "PAY_PER_REQUEST"

### Service configuration (serverless.yml) content

```yaml
N/A
```

### Command name and used flags

sls deploy --stage dev

### Command output

```shell
CREATE_FAILED: MktgHasOffersLog (AWS::DynamoDB::Table)
Resource handler returned message: “User: arn:aws:sts::924149985142:assumed-role/codebuildv5-CodeBuildStandardRole-1DG17IWSQD2WT/AWSCodeBuild-482aa296-5eb2-4041-a4c9-6f9a78942f67 is not authorized to perform: dynamodb:DescribeTable on resource: arn:aws:dynamodb:us-east-1:924149985142:table/Mktg-HasOffers-Logs-dev because no identity-based policy allows the dynamodb:DescribeTable action (Service: DynamoDb, Status Code: 400, Request ID: D2M4SJM8T0KA7ETBU5FKS224M3VV4KQNSO5AEMVJF66Q9ASUAAJG)” (RequestToken: ecf708b3-58eb-a783-8775-890035cf6fc3, HandlerErrorCode: AccessDenied)
```

### Environment information

```shell
3.27.0
```

**Comment by medikoo** [comment]
@jonathanalberghini issue is described in the error message you received from AWS. This problem is not influenced in any way by the Serverless Framework. I'm moving this to discussions
